pkg/controllers/vdb/verticadb_controller.go

```diff
@@ -131,11 +132,17 @@ func (r *VerticaDBReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	pfacts := MakePodFacts(r, prunner)
 	dispatcher := r.makeDispatcher(log, vdb, prunner, passwd)
 	var res ctrl.Result
-
+	const UninstallReconciler = "UninstallReconciler"
 	// Iterate over each actor
 	actors := r.constructActors(log, vdb, prunner, &pfacts, dispatcher)
 	for _, act := range actors {
-		log.Info("starting actor", "name", fmt.Sprintf("%T", act))
+		actName := fmt.Sprintf("%T", act)
```

Comment:
I realize this is WIP, but we should just add the `UseVClusterOps` check right in `uninstall_reconciler.go`. It is fine to call that and perform a no-op for vclusterops deployments.

Reply:
Yes, like we have done for `ScheduleOnly` in `(s *UninstallReconciler) Reconcile()`